client/src/routes/fight/bar/FightTime.svelte

```diff
@@ -81,11 +84,18 @@
 </script>
 
 <div>
-    <button class="returnButton" on:click={() => dispatch('return')} hidden={$emptySeq}>
-        <Icon icon="bx:left-arrow-circle" color="#2f4858" height="2rem"/>
-    </button>
-
-  <div class="info">
+  <button
+    class="returnButton"
+    on:click={() => dispatch('return')}
+    hidden={$emptySeq}
+  >
```

Comment:
Możemy rozważyć dodanie takiego sposóbu przenoszenia atrybutów komponentu do prettiera - faktycznie poprawia czytelność

Reply:
Jeżeli dobrze kojarzę, to są to domyślne ustawienia, nieznacznie zmienione w ustawieniach w `.prettierrc`